logs-drain-remove panic if no args is provided

```
scalingo -a john-test log-drains-remove 
panic: runtime error: index out of range [0] with length 0

goroutine 1 [running]:
github.com/Scalingo/cli/cmd.glob..func156(0xc0000e2c60)
        /home/john/gocode/src/github.com/Scalingo/cli/cmd/log_drains.go:181 +0x5dc
github.com/Scalingo/cli/cmd.(*AppCommands).addCommand.func1(0xc0000e2c60)
        /home/john/gocode/src/github.com/Scalingo/cli/cmd/commands.go:61 +0x1f3
github.com/Scalingo/cli/vendor/github.com/urfave/cli.HandleAction(0xa5dfa0, 0xc000141480, 0xc0000e2c60, 0xc0000e2c60, 0x0)
        /home/john/gocode/src/github.com/Scalingo/cli/vendor/github.com/urfave/cli/app.go:525 +0x127
github.com/Scalingo/cli/vendor/github.com/urfave/cli.Command.Run(0xb76368, 0x11, 0x0, 0x0, 0x0, 0x0, 0x0, 0xb936cf, 0x40, 0x0, ...)
        /home/john/gocode/src/github.com/Scalingo/cli/vendor/github.com/urfave/cli/command.go:174 +0x51c
github.com/Scalingo/cli/vendor/github.com/urfave/cli.(*App).Run(0xc0001861c0, 0xc0000c0120, 0x6, 0x6, 0x0, 0x0)
        /home/john/gocode/src/github.com/Scalingo/cli/vendor/github.com/urfave/cli/app.go:276 +0x718
main.main()
        /home/john/gocode/src/github.com/Scalingo/cli/scalingo/main.go:157 +0x700
```

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -6,6 +6,8 @@
   apps [#588](https://github.com/Scalingo/cli/pull/588)
 * Fetch the SSH Login endpoint from our region metadatas
   [#592](https://github.com/Scalingo/cli/pull/592)
+* Fix a panic when logs-drain-remove was launched without any args
+  [#595](https://github.com/Scalingo/cli/pull/595)
 
 ### 1.19.0
 

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -6,7 +6,7 @@
   apps [#588](https://github.com/Scalingo/cli/pull/588)
 * Fetch the SSH Login endpoint from our region metadatas
   [#592](https://github.com/Scalingo/cli/pull/592)
-* Fix a panic when logs-drain-remove was launched without any args
+* Fix a panic when log-drains-remove was launched without any args
   [#595](https://github.com/Scalingo/cli/pull/595)
 
 ### 1.19.0

--- a/cmd/log_drains.go
+++ b/cmd/log_drains.go
@@ -165,6 +165,11 @@ var (
 
 		Action: func(c *cli.Context) {
 			currentApp := appdetect.CurrentApp(c)
+			if len(c.Args()) != 1 {
+				cli.ShowCommandHelp(c, "log-drains-remove")
+				return
+			}
+			drain := c.Args()[0]
 
 			var addonID string
 			if c.GlobalString("addon") != "<addon_id>" {
@@ -178,7 +183,7 @@ var (
 				return
 			}
 
-			message := "This operation will delete the log drain " + c.Args()[0]
+			message := "This operation will delete the log drain " + drain
 			if addonID == "" && !c.Bool("only-app") {
 				// addons + app
 				message += " for the application and all its addons"
@@ -195,16 +200,11 @@ var (
 				return
 			}
 
-			var err error
-			if len(c.Args()) == 1 {
-				err = log_drains.Remove(currentApp, log_drains.RemoveAddonOpts{
-					AddonID: addonID,
-					OnlyApp: c.Bool("only-app"),
-					URL:     c.Args()[0],
-				})
-			} else {
-				cli.ShowCommandHelp(c, "log-drains-remove")
-			}
+			err := log_drains.Remove(currentApp, log_drains.RemoveAddonOpts{
+				AddonID: addonID,
+				OnlyApp: c.Bool("only-app"),
+				URL:     drain,
+			})
 
 			if err != nil {
 				errorQuit(err)

